.travis.yml

```diff
@@ -7,25 +8,24 @@ notifications:
   email:
     on_success: never
     on_failure: change
-  slack: bsf-bots:RDGCoMDDxnlCvnUfvhq8pQ36
+    slack: bsf-bots:RDGCoMDDxnlCvnUfvhq8pQ36
```

Comment:
You can remove this notification for slack as we no longer use this.

Reply:
Done.